debian/patches/0051-add-coreimage-based-vf-transpose-vt-filter.patch

```diff
@@ -0,0 +1,573 @@
+Index: FFmpeg/configure
+===================================================================
+--- FFmpeg.orig/configure
++++ FFmpeg/configure
+@@ -3934,7 +3934,7 @@ tonemap_vaapi_filter_deps="vaapi VAProcF
+ tonemap_opencl_filter_deps="opencl const_nan"
+ transpose_opencl_filter_deps="opencl"
+ transpose_vaapi_filter_deps="vaapi VAProcPipelineCaps_rotation_flags"
+-transpose_vt_filter_deps="videotoolbox VTPixelRotationSessionCreate"
++transpose_vt_filter_deps="coreimage videotoolbox"
+ transpose_vulkan_filter_deps="vulkan spirv_compiler"
+ unsharp_opencl_filter_deps="opencl"
+ uspp_filter_deps="gpl avcodec"
+Index: FFmpeg/libavfilter/vf_transpose_vt.c
+===================================================================
+--- FFmpeg.orig/libavfilter/vf_transpose_vt.c
```

Comment:
I'm going to make a version merges two of them. But this is good enough for now.